app/lib/katello/resources/candlepin/consumer.rb

```diff
@@ -102,8 +102,7 @@ def refresh_entitlements(uuid)
             self.post(join_path(path(uuid), 'entitlements'), "", self.default_headers).body
           end
 
-          def consume_entitlement(uuid, pool, quantity = nil)
-            uri = join_path(path(uuid), 'entitlements') + "?pool=#{pool}"
+          def consume_entitlement(_uuid, _pool, quantity = nil)
             uri += "&quantity=#{quantity}" if quantity && quantity > 0
```

Comment:
`s/+=/=/` ?

Reply:
i think that line was deleted by mistake...